tests/src/log4net.Tests.csproj

```diff
@@ -19,245 +18,25 @@
  under the License.
 
 -->
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectType>Local</ProjectType>
-    <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{B0530F10-0238-49A9-93B0-8EF412E90BCF}</ProjectGuid>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ApplicationIcon>
-    </ApplicationIcon>
-    <AssemblyKeyContainerName>
-    </AssemblyKeyContainerName>
+    <!--
+         Currently multitargeting tests doesn't work, separate projects are required.
+         (tracked at https://github.com/Microsoft/vstest/issues/298 and https://github.com/Microsoft/vstest/issues/624)
+    -->
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Actually, there should be one and only one way to build the project and that way should be used by developers and CI. Right now we have nant and it looks like latest microsoft/dotnet docker image bases on debian stretch which no longer comes with nant. We could work around that for now, but my impression is that we should move away from nant as soon as possible. To me it looks like it is no longer maintained and we should no longer invest into a sinking ship. This means that ideally we would have one project file that builds all targets and another one that tests all targets. Now testing all targets does not work right now, so we need multiple projects for that usecase.